Cargo.toml

```diff
@@ -14,4 +14,4 @@ anyhow = "1.0.32"
 serde_json = "1.0.57"
 parking_lot = "0.11.0"
 warp = "0.2"
-tokio = { version = "0.2.4", features = ["macros", "test-util"] }
\ No newline at end of file
+tokio = { version = "0.2.4", features = ["macros", "test-util"] }
```

Comment:
Do we really want to depend on tokio? We used `async-std` in dex-services and it worked fine. Tokio is a big hammer if I recall correctly. How come we already depend on it?